README.rst

```diff
@@ -1291,8 +1295,8 @@ Choose from the list of available rules:
     ``true``
   - ``identical`` (``bool``, ``null``): change identical (``===``, ``!==``) statements;
     defaults to ``true``
-  - ``less_and_greater`` (``bool``, ``null``): change less and greater than (``<``, ``<=``,
-    ``>``, ``>=``) statements; defaults to ``null``
+  - ``less_and_greater`` (``bool``, ``null``): change less and greater than (``<``,
+    ``<=``, ``>``, ``>=``) statements; defaults to ``null``
```

Comment:
This is the result of escaping before doing the word wrap. So after the escaping the line it gets longer, than it gets word wrapped and then unescaped later on.
It took me way to long to figure out why this happened and wonder if it worth fixing it.

Reply:
it's not worth spending time to fix it. I don't consider it a bug. both outcomes are fine :+1: